### ISSUE TITLE
Support some way to set the starting monitor

### ISSUE THREAD

**Issue by ericoporto** [adventuregamestudio/ags#2280]
**Describe the problem**
AGS by default enters full-screen in the primary monitor (on Windows), but it may be useful to have it full-screen in a different monitor - either because the user has the primary monitor in vertical, or the secondary monitor is a TV intended for gaming, or even if the user is developing the game in the AGS Editor and has it open in the primary monitor and wants to debug the game in a secondary monitor.

**Suggested change**
Some config in acsetup.cfg/command-line/winsetup that tells the game to prefer a different monitor, in a way that if such monitor doesn't exist, it still fallback to the default behavior of using the primary monitor.

This is only meant for it to be the initial monitor, currently it's possible to move the window to a different monitor and when it enters in full-screen the window does full-screen in the same monitor as it was.

If a command-line option is added, it would be nice to have a setting in the Editor Preferences that you can set and would pass the preference when testing the game.

Note:  I am only aggregating previous suggestions and questions from others, I only have 1 monitor and will take me a bit to test things for this if someone comes up with a solution.

**Comment by ivan-mogilko** [comment]
I guess this is mostly a question of identification (number or string). IIRC SDL supports telling which monitor to create window on.

**Comment by ericoporto** [comment]
Yes, from what I can tell SDL uses an index to point to the monitor. It also has an API to get a display name (for showing somewhere) but I think there isn't an easy way to show this in either ags command line or winsetup. So my guess is any API would just use the number as some index for the display - and probably be like 0/auto/empty is default and 1 is index 0, 2 index 1 and so on.

https://wiki.libsdl.org/SDL2/SDL_GetNumVideoDisplays
https://wiki.libsdl.org/SDL2/SDL_GetDisplayName
https://wiki.libsdl.org/SDL2/SDL_GetDisplayBounds

**Comment by ivan-mogilko** [comment]
> but I think there isn't an easy way to show this in either ags command line or winsetup

You can show this in winsetup, just like it shows audio drivers from SDL, for example.

**Comment by ivan-mogilko** [comment]
I have two monitors at my working place now so i would be able to work on this being able to test immediately.
